code/game/objects/items/cards_ids.dm

```diff
@@ -835,6 +835,12 @@
 /obj/item/card/id/proc/get_trim_sechud_icon_state()
 	return trim?.sechud_icon_state || SECHUD_UNKNOWN
 
+/obj/item/card/id/interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
+	if(iscash(interacting_with))
+		insert_money(interacting_with, user)
+		return ITEM_INTERACT_SUCCESS
```

Comment:
Technically this interaction isn't always a success, cause inserting money can fail, so I feel it'd be better to do:
```suggestion
		return insert_money(tool, user) ? ITEM_INTERACT_SUCCESS : ITEM_INTERACT_BLOCKING
```
Much like the `item_interaction(...)` part of the code does
https://github.com/tgstation/tgstation/blob/c28477076b4d566b04c43c8827a64396312daf6a/code/game/objects/items/cards_ids.dm#L559-L560